test/capi/testLinearGradient.cpp

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright (c) 2020-2021 Samsung Electronics Co., Ltd. All rights reserved.
+
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+#include <stdint.h>
+#include <thorvg_capi.h>
+#include "catch.hpp"
+
+TEST_CASE("Basic Create", "[capi-radial-gradient]")
+{
+    Tvg_Gradient *gradient = NULL;
+    gradient = tvg_linear_gradient_new();
+    REQUIRE(gradient != NULL);
+    REQUIRE(tvg_gradient_del(gradient) == TVG_RESULT_SUCCESS);
+}
+
+TEST_CASE("Set gradient start and end position", "[capi-radial-gradient]")
+{
+    Tvg_Gradient *gradient = NULL;
+    float x1 = 0.0, y1 = 0.0, x2 = 0.0, y2 = 0.0;
+
+    gradient = tvg_linear_gradient_new();
+
+    REQUIRE(gradient != NULL);
+    REQUIRE(tvg_linear_gradient_set(gradient, 10.0, 10.0, 50.0, 50.0) == TVG_RESULT_SUCCESS);
+    
```

Comment:
I would use different values for each arg so we could catch mistakes like switching the variables (ex x1 with y1 etc)